cmc-domain-model/src/main/java/uk/gov/hmcts/cmc/claimstore/constraints/AgeRangeConstraintValidator.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.cmc.claimstore.constraints;
```

Comment:
This is claim data model so `claimstore` does not match here. It should be something like `uk.gov.hmcts.cmc.case.constraints`. I am happy for you to do it in separate PR if you prefer it.

Reply:
I agree, and will do in a separate PR.